docs/GCS-batchsink.md

```diff
@@ -26,6 +26,12 @@ Properties
 ----------
 **Reference Name:** Name used to uniquely identify this sink for lineage, annotating metadata, etc.
 
+**Use Connection** Whether to use a connection, if a connection is used,
```

Comment:
@itsankit-google Can we rewrite as "Whether to use a connection. If a connection is used,"